### PR TITLE
Add --sidebar option to create:controller

### PR DESCRIPTION
**Pull request by mjauvin**
Add an option to artisan create:controller to use new specific create/update stubs with a sidebar for the secondaryTabs fields (like for the backend users controller)

### CODE REVIEW

File: modules/backend/console/scaffold/controller/controller.stub [hunk 1]
```diff
@@ -21,5 +21,7 @@ class {{ studly_name }} extends Controller
         parent::__construct();
 
         BackendMenu::setContext('{{ plugin_code }}', '{{ lower_plugin }}', '{{ lower_name }}');
+
+        $this->bodyClass = '{{ body_class }}';
```
Comment (LukeTowers) [inline modules/backend/console/scaffold/controller/controller.stub]: This should only be added if the value has been set

File: modules/backend/console/CreateController.php [hunk 2]
```diff
@@ -69,6 +67,13 @@ protected function prepareVars(): array
             $model = Str::singular($vars['name']);
         }
         $vars['model'] = $model;
+        $vars['body_class'] = $this->option('sidebar') ? 'compact-container' : '';
+
+        $suffix = $this->option('sidebar') ? '-sidebar' : '';
+
+        $this->stubs["scaffold/controller/create{$suffix}.stub"] = 'controllers/{{lower_name}}/create.php';
+        $this->stubs["scaffold/controller/update{$suffix}.stub"] = 'controllers/{{lower_name}}/update.php';
+        $this->stubs["scaffold/controller/preview{$suffix}.stub"] = 'controllers/{{lower_name}}/preview.php';
```
Comment (LukeTowers) [inline modules/backend/console/CreateController.php]: Can we have the stubs be in a folder by style / layout? i.e: `controller/[standard|sidebar]/create.stub`

Comment (mjauvin) [reply modules/backend/console/CreateController.php]: That would mean duplicating all the other stubs that stay the same for both layouts?

Comment (mjauvin) [reply modules/backend/console/CreateController.php]: Or do you mean only for the create/update/preview stubs?

Comment (LukeTowers) [reply modules/backend/console/CreateController.php]: Just the layout specific ones

File: modules/backend/console/CreateController.php [hunk 3]
```diff
@@ -44,14 +45,11 @@ class CreateController extends BaseScaffoldCommand
...
-        'scaffold/controller/create.stub'        => 'controllers/{{lower_name}}/create.php',
-        'scaffold/controller/index.stub'         => 'controllers/{{lower_name}}/index.php',
-        'scaffold/controller/preview.stub'       => 'controllers/{{lower_name}}/preview.php',
-        'scaffold/controller/update.stub'        => 'controllers/{{lower_name}}/update.php',
-        'scaffold/controller/controller.stub'    => 'controllers/{{studly_name}}.php',
+        'scaffold/controller/_list_toolbar.stub'   => 'controllers/{{lower_name}}/_list_toolbar.php',
+        'scaffold/controller/config_form.stub'     => 'controllers/{{lower_name}}/config_form.yaml',
+        'scaffold/controller/config_list.stub'     => 'controllers/{{lower_name}}/config_list.yaml',
+        'scaffold/controller/index.stub'           => 'controllers/{{lower_name}}/index.php',
+        'scaffold/controller/controller.stub'      => 'controllers/{{studly_name}}.php',
```
Comment (LukeTowers) [inline modules/backend/console/CreateController.php]:
```suggestion
        'scaffold/controller/_list_toolbar.stub' => 'controllers/{{lower_name}}/_list_toolbar.php',
        'scaffold/controller/config_form.stub'   => 'controllers/{{lower_name}}/config_form.yaml',
        'scaffold/controller/config_list.stub'   => 'controllers/{{lower_name}}/config_list.yaml',
        'scaffold/controller/index.stub'         => 'controllers/{{lower_name}}/index.php',
        'scaffold/controller/controller.stub'    => 'controllers/{{studly_name}}.php',
```

**Pull request merged**